src/tests/stringy.spec.ts

```diff
@@ -6,7 +6,8 @@ describe('Stringy methods', () => {
   describe('isPrefix', () => {
     const nougat = new Stringy('nougat');
     const NOUgat = new Stringy('NOUgat');
-    const nougatWithTrailingSpace = new Stringy('nougat ')
+    const nougatWithTrailingSpace = new Stringy('nougat ');
+    const choco = new Stringy('choco');
```

Comment:
Codacy has a fix for the issue: [Replace `'choco'` with `"choco"`](https://app.codacy.com/gh/VictorMaria/Strings-and-Arrays/pullRequest?prid=9807711)
```suggestion
    const choco = new Stringy("choco");
```